frontend/src/metabase/public/containers/PublicAction/PublicAction.unit.spec.tsx

```diff
@@ -184,17 +184,14 @@ describe("PublicAction", () => {
     expect(screen.queryByRole("form")).not.toBeInTheDocument();
   });
 
-  it("immediately executes an action without parameters", async () => {
+  it("does not immediately executes an action without parameters", async () => {
```

Comment:
Typo:

```suggestion
  it("does not immediately execute an action without parameters", async () => {
```

Or maybe it's worth to be re-written:

```suggestion
  it("handles actions without parameters", async () => {
```